client/src/pages/Lobby.js

```diff
@@ -100,6 +106,12 @@ const Lobby = () => {
   };
 
   const addUserAndQuestions = async () => {
+    const {
+      data: { interview },
+    } = await getInterview(roomId);
+    const creator = interview.users[0].user._id;
+    setCreatorId(creator);
+
```

Comment:
Because set state is async, it's probably safer to use `if (creator)` below vs `creatorId`

Reply:
Good catch! I'm going to fix this on my persist-code branch and then merge that one into dev

Reply:
If that one covers everything that this one does, you can go ahead and close this PR